### ISSUE TITLE
Resource Reload Failed with Oculus 1.7.0 in heavily modded instance

### ISSUE THREAD

**Issue by katubug** [Asek3/Oculus#609]
### What happened?

When loading the game in my in-dev modpack, the newest version of Oculus prevents resources from reloading properly, which breaks all mod resources, resourcepacks, etc. The game does boot up to the main menu, but with FancyMenu completely broken. I have not attempted to load a world in this state.

### Screenshots

https://i.imgur.com/bY2A2pv.png

### Relevant log output

```shell
Unsure, attached full log below.
```

### Minecraft Version

Minecraft 1.20.1

### Oculus Version

1.7.0

### Rubidium / Embeddium Version

embeddium 0.3.14

### Operating System

Windows 11

### What is your GPU?

Geforce RTX 3060 (Laptop)

### Java Version

Java 17

### Additional context

[latest.log](https://github.com/Asek3/Oculus/files/15153326/latest.log)

**Comment by katubug** [comment]
I wanted to add some additional information. The same thing occurs when I install WorldEdit 7.2.15, so I did a little more testing, and found that when I disable Create, Create Addons, and Oculus/Flywheel compat, the game loads fine, even with WorldEdit and with the most recent Oculus. I realized there were a few mentions of Create's "`com.simibubi.create.content.kinetics.fan.processing.FanProcessingTypeRegistry`", which is what inspired me to test without the mod.

Is this something I should be reporting to Create?
